### PR TITLE
skip unit stride consistency check if the dimension size is 1

### PR DESCRIPTION
**Pull request by anstaf**

### CODE REVIEW

**Comment by havogt** [comment]
launch jenkins

File: include/gridtools/storage/adapter/python_sid_adapter.hpp [hunk 1]
```diff
@@ -54,7 +54,25 @@ namespace gridtools {
...
+            return std::forward<Strides>(strides);
+        }
+
+        template <size_t UnitStrideDim,
+            class Strides,
+            class Shape,
+            std::enable_if_t<(UnitStrideDim < tuple_util::size<std::decay_t<Strides>>::value), int> = 0>
+        decltype(auto) assign_unit_stride(Strides &&strides, Shape &&shape) {
+            return tuple_util::transform_index(
+                transform_strides_f<UnitStrideDim>{tuple_util::get<UnitStrideDim>(std::forward<Shape>(shape)) > 1},
```
Comment (havogt) [inline include/gridtools/storage/adapter/python_sid_adapter.hpp:71]: Let's add a comment here with link to `https://numpy.org/devdocs/release/1.8.0-notes.html#npy-relaxed-strides-checking`

Comment (anstaf) [reply include/gridtools/storage/adapter/python_sid_adapter.hpp]: done

**Review by havogt — APPROVED** [review]

**Comment by havogt** [comment]
test ok (except broken hip setup)

**Pull request merged**